frontend/src/components/pages/HomePage.css

```diff
@@ -0,0 +1,8 @@
+#homepage-display-state-toggle-section {
+    background-color: #FAFAFA;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    min-width: 800px;
+    width: 60%;
+    /* TODO who should manage the width of story cards - should be same width as this section */
```

Comment:
components take on the width of their parent component by default. To prevent any inconsistency, you can wrap both the header and the list of story cards in the same div, and set the width of the div manually (eg `width: 60%;`)